x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/span_flyout/span_db.tsx

```diff
@@ -76,9 +31,11 @@ export function SpanDatabase({ spanDb }: Props) {
           )}
         </h3>
       </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiCodeBlock language="sql" fontSize="m" paddingSize="m">
+        {spanDb.statement}
```

Comment:
Do you mind looking if this bug https://github.com/elastic/kibana/issues/127063 is fixed by changing to this component?